fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2985,7 +2985,7 @@ public void alterMaterializedView(AlterMaterializedViewStatement stmt) throws Dd
     }
 
     @Override
-    public void refreshMaterializedView(String dbName, String mvName) throws DdlException, MetaNotFoundException {
+    public void refreshMaterializedView(String dbName, String mvName, int priority) throws DdlException, MetaNotFoundException {
```

Comment:
why not use TaskRunPriority as argument?

Reply:
The priority is not only those of the enumeration values, there may be more complicated situations later, so it should be of the int type, providing the enumeration value is just a simple template.